ui/admin/config/environment.js

```diff
@@ -56,62 +56,84 @@ module.exports = function (environment) {
     oidcPollingTimeoutSeconds: 1,
 
     documentation: {
-      baseURL: 'https://boundaryproject.io/help/admin-ui',
+      baseURL: 'https://developer.hashicorp.com',
```

Comment:
I'm fine with this as the baseURL but will they always start with `/boundary/docs`?

Reply:
all the links we have rn start with `/boundary` but not `/docs` some are `/boundary/tutorials`

Reply:
Gotcha, yeah I wonder if we should assume `/boundary` as part of the base URL but it's not a big deal either way

Reply:
ah okay im looking at past feedback and realized that i misunderstood something. Ill change this back to use `/boundary`. good catch thank you!